bin/find-lib-in-path.py

```diff
@@ -36,16 +35,17 @@ def get_ld_paths():
     # - LD_LIBRARY_PATH
     # - /etc/ld.so.cache (instead we will parse /etc/ld.so.conf)
     # - /lib, /usr/lib (or maybe /lib64, /usr/lib64)
-    paths = []
-    if "LD_LIBRARY_PATH" in os.environ:
-        paths.extend(os.environ["LD_LIBRARY_PATH"].split(":"))
+    PREFIX = os.getenv("PREFIX") # Termux & etc.
+    LDPATH = os.getenv("LD_LIBRARY_PATH") 
+    paths = ["/lib", "/usr/lib", "/lib64", "/usr/lib64"]
+    if LDPATH: 
+        paths.extend(LDPATH.split(":"))
+    if PREFIX:
+        paths.extend([PREFIX, PREFIX + "/lib", PREFIX + "/usr/lib", PREFIX + "/lib64", PREFIX + "/usr/lib64"])
     if os.path.exists("/etc/ld.so.conf"):
```

Comment:
Btw, maybe you also want to check the same for `f"{PREFIX}/etc/ld.so.conf"`?

Reply:
Indeed, thanks for the catch, i completely ignored it

Reply:
* Removed PREFIX
* Added check for `PREFIX +  "/etc/ld.so.conf"`